src/components/floweditor/FlowEditor.tsx

```diff
@@ -146,7 +147,24 @@ export const FlowEditor = () => {
 
   if (flowName && flowName.flows.length > 0) {
     flowTitle = flowName.flows[0].name;
-    flowKeywords = flowName.flows[0].keywords.join(', ');
+    const keywords = flowName.flows[0].keywords;
+    if (keywords.length < 8) {
```

Comment:
It might be better to restrict this based on the string length that's rendered rather than the number of words.